packages/styles/scss/patterns/leadspace/_leadspace.scss

```diff
@@ -87,6 +119,12 @@
     }
   }
 
+  .#{$prefix}--leadspace--g100 {
+    @include carbon--theme($carbon--theme--g100) {
+      @include themed-items;
+    }
+  }
+
```

Comment:
![greatsuccess](https://user-images.githubusercontent.com/1641214/69459974-f6542880-0d40-11ea-851e-2bd1539ffe98.gif)

Is there a default in case the adopter doesn't include the modifier class?

Reply:
you mean if they don't pick the `g100` theme? it defaults to the white theme